migrations/Version20241211222236.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20241211222236 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE reset_password_request CHANGE selector selector VARCHAR(20) NOT NULL');
```

Comment:
why are you touching reset password in this task?

Reply:
It generated that line on command make migration, and i deleted that line in up and also in down schema but i didnt save it...

Reply:
I've just looked at the task implementation and the database, and in normal usage, the selector doesn't cause any issues, but we are not using it.